Gemfile

```diff
@@ -11,11 +11,17 @@ group :development do
 end
 
 group :test do
```

Comment:
According to the [README](https://github.com/guard/guard#installation) `gem 'guard'` belongs to the `:development` group. In Cubki's `Gemfile` it's also in `:development`.
-> Is there a difference if we put it into `:test`?

Reply:
Ah, sorry my mistake. It would better to add to `:development` group. Thank you!